src/engine/SCons/Tool/packaging/__init__.xml

```diff
@@ -62,24 +62,25 @@ option or with the &cv-PACKAGETYPE; construction variable. Currently
 the following packagers available:
 </para>
 
-<para>
- * msi - Microsoft Installer
- * rpm - RPM Package Manger
- * ipkg - Itsy Package Management System
- * tarbz2 - bzip2 compressed tar
- * targz - gzip compressed tar
- * tarxz - xz compressed tar
- * zip - zip file
- * src_tarbz2 - bzip2 compressed tar source
- * src_targz - gzip compressed tar source
- * src_tarxz - xz compressed tar source
- * src_zip - zip file source
-</para>
+<para><literal>msi</literal> - Microsoft Installer</para>
```

Comment:
Should this be some kind of list instead?

Reply:
nothing else really seemed to work without putting in a pointless amount of whitespace.  yes, it should be a list with the "compact" attribute, but the joy of markup languages is they take the low-level rendering decisions out of your hands. Bring back troff!!!!